examples/mixins/simple/simple.js

```diff
@@ -52,18 +32,21 @@ const docMixin = {
   extend: {
     /**
     * Simple tweeting function.
+    * @returns {Promise<Object|null>} A promise that when resolved means this mixin method
```

Comment:
It tries to clarify that you don't need to resolve the promise with a value, I'll change these to simply `Promise` to be consistent with other examples.